### PR TITLE
Feature/alert new post

### PR DESCRIPTION
**Pull request by dombioinfo**
Add blink effect when new message is received

### CODE REVIEW

**Comment by pypingou** [comment]
Maybe we should remove all the commented console.log directly, I'm not sure there is much interest in keeping them in.

File: js/fedora_apps.js [hunk 1]
```diff
@@ -182,5 +182,39 @@ function setup_websocket_listener() {
         // the websocket connection, but we have to go query again to
         // get the fedmsg.meta information.
         update_fedmsg(id_lookup[category], category, deploy);
+        
+        // Add blink effect on page's title to alert user of new post
+        // Note: this code is adapted from canop's blog (canop.org)
```
Comment (pypingou) [inline js/fedora_apps.js]: Might be worth putting the whole url.

Is there a license on the code there?

File: js/fedora_apps.js [hunk 2]
```diff
@@ -142,14 +142,14 @@ function setup_websocket_listener() {
...
-        console.log("onclose");
+//        console.log("onclose");
 //        socket=null;
         setup_websocket_listener();
     };
 
     // Our main callback
     socket.onmessage = function(e){
-        console.log("onmessage");
+//        console.log("onmessage");
```
Comment (pypingou) [inline js/fedora_apps.js]: I think we should just remove these lines rather than commenting them out

File: js/fedora_apps.js (hunk 1 above)
Comment (dombioinfo) [inline js/fedora_apps.js]: In fact, canop re-writes in his "words" what it is explained on page : https://developer.mozilla.org/en-US/docs/Web/Guide/User_experience/Using_the_Page_Visibility_API?redirectlocale=en-US&redirectslug=DOM%2FUsing_the_Page_Visibility_API which license is "Creative Commons Attribution-ShareAlike".

Comment (pypingou) [inline js/fedora_apps.js]: It's actually public license:

```
Code samples added on or after August 20, 2010 are in the public domain. No licensing notice is necessary, but if you need one, you can use: "Any copyright is dedicated to the Public Domain. http://creativecommons.org/publicdomain/zero/1.0/".
```

So from a license point of view, it's ok to merge it, just add the full link please.

File: js/fedora_apps.js [hunk 3]
```diff
@@ -182,5 +174,38 @@ function setup_websocket_listener() {
...
+        })();
+
+        var blinkTimer = null;
+        visibility(function(){
+            if (visibility()) {
+                clearInterval(blinkTimer);
+            } else {
+                blinkTimer = setInterval(function() {
+                    var title = document.title;
+                    document.title = (title === originalTitle) ? "New " + id_lookup[category] : originalTitle;
```
Comment (pypingou) [inline js/fedora_apps.js:206]: What about moving the test up a little so that we do no update the document.title unless there is a change?

Comment (dombioinfo) [inline js/fedora_apps.js:206]: This effect is applied at each received message (from all categories). Perhaps we can apply this effect only when we receive messages which its categroy is 'planet'.

Comment (pypingou) [inline js/fedora_apps.js:206]: I think it's worth keeping it for all pages

**Comment by pypingou** [comment]
I have this running for a while now and it works good, so let's merge it :)

**Pull request merged**